week-1/L-functions-nested/exercise2.js

```diff
@@ -3,3 +3,17 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function Capital_Letters() {
```

Comment:
Function names in JavaScript should be in camelCase. This means it should be `function capitalLetters() {` instead of `function Capital_Letters() {`.